axon-framework/queries/query-dispatchers.md

```diff
@@ -227,5 +227,132 @@ commandGateway.sendAndWait(new RedeemCmd("gc1", amount));
 
    Since we subscribed to updates with the `println()` method, the update will be printed out once it is received.
 
+
+### Streaming queries
```

Comment:
I'd expect the streaming query to be added to the top of this file in the "The Query Bus and Query Gateway".
We summarize the other types, so why not add the streaming query there too?